src/AVFoundation/Enums.cs

```diff
@@ -718,11 +718,13 @@ public enum AVPlayerTimeControlStatus : nint
 		Playing
 	}
 
-	[NoWatch, NoTV, iOS (10,0), Mac (10,12)]
+#if !MONOMAC || !XAMCORE_4_0
```

Comment:
APIDiff claimed it was an API removal when i tested it. But as you point out, that doesn't make any sense. Let me validate against 2.10 and fix.